installer/scripts/omsadmin.sh

```diff
@@ -796,6 +820,37 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    local result_limit="No Limit"
+    if [ ! -f $PROC_LIMIT_CONF ]; then
+        log_error "Limits configuration file '$PROC_LIMIT_CONF' missing!"
+        return 1
+    fi
+    local count=`get_current_omsagent_proc_limit | wc -l`
+
+    if [ "$count" -eq "0" ]; then
+        log_info "There is NO present limit to number of processes for $AGENT_USER."
+    else
+        if [ "$count" -gt "1" ]; then
+            log_info "$count $AGENT_USER entries found in $PROC_LIMIT_CONF."
+            log_info ">>Last entry should apply<<" # NOTE:  This will not pick up -1 entries.
+            get_current_omsagent_proc_limit
+            log_info "End of the $count duplicate entry info Message"
+        fi
+        result_limit=`get_current_omsagent_proc_limit | tail -1 | awk '{print $4}'`
+        if [ "$result_limit" -eq "-1" ]; then
+            result_limit="No Limit"
+            #log_info "There is NO present limit to number of processes for $AGENT_USER."
```

Comment:
You should delete this line entirely instead of just commenting it out.

Reply:
Got it.
I don't get a comment for the item on find_current_*() for some reason.  I'll take it out as your question implies it is not yet used outside the script.  Thanks.